src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const mainImage = document.getElementById('largeImg');
+const gallery = document.querySelector('.gallery__list');
+
+gallery.addEventListener('click', (evnt) => {
+  evnt.preventDefault();
+
+  const linkToSmall = evnt.target.closest('a');
```

Comment:
Ensure that 'evnt.target.closest('a')' correctly identifies the anchor element. If the click event is triggered on a child element within the anchor, 'closest' will still return the anchor element, which is correct. However, if the structure of the HTML changes, this might not work as intended. Consider checking if 'evnt.target' is an image or another element that should trigger the event.